Gemfile

```diff
@@ -38,6 +38,7 @@ gem "dry-struct", "~> 1.0" # Typed structs and value objects
 gem "email_validator", "~> 2.0" # Email validator for Rails and ActiveModel
 gem "emoji_regex", "~> 2.0" # A pair of Ruby regular expressions for matching Unicode Emoji symbols
 gem "envied", "~> 0.9" # Ensure presence and type of your app's ENV-variables
+gem "fast_jsonapi", "~> 1.5" # Serializer for Ruby objects
```

Comment:
Can you test with the `Oj` gem as well? [Fast-jsonapi can use it as a backend](https://github.com/Netflix/fast_jsonapi/blob/81375cfcf751a76bcd7da4927a58375b03772144/lib/fast_jsonapi/multi_to_json.rb#L56) which is definitely faster than Ruby's JSON that it's used by default. You just have to include it: https://rubygems.org/gems/oj

I'm only a little uncomfortable about the fact that we're going to have another form of serialization right now (to json, jbuilder and this)...

Reply:
I'll look at that. I'm not 100% comfortable with having a lot of ways to serialize too, but I was also uncomfortable with using `jbuilder` in this case.